modules/Quiz/resources/quiz.js

```diff
@@ -559,9 +559,18 @@
                 .click(function () {
                     $(".close-button")
                     .click(function () {
-                        interfaceElement.find(".open-question-textarea").val(holdAnswer);
-                        interfaceElement.find(".open-question-chars .chars").text(holdAnswer.length);
-                        if (holdAnswer.length !== 0) {
+                        if (!cPo.isAnswerd) {
+                            interfaceElement.find(".open-question-textarea").val(holdAnswer);
+                            interfaceElement.find(".open-question-chars .chars").text(holdAnswer.length);
+                            if (holdAnswer.length !== 0) {
+                                interfaceElement.find("#open-question-clear,#open-question-save").removeAttr("disabled");
+                            }
+                        } else {
+                            interfaceElement.find(".open-question-textarea").val(holdAnswer);
+                            interfaceElement.find(".open-question-chars .chars").text(holdAnswer.length);
```

Comment:
same method on 563 and 569, on 564 and 570.
we can move them to own function, like
```
function setAnswer(answer) {
     interfaceElement.find(".open-question-textarea").val(answer);
     interfaceElement.find(".open-question-chars .chars").text(answer.length);
}
```